package.json

```diff
@@ -1,12 +1,12 @@
 {
-  "name": "web3-plugin-template",
+  "name": "web3-plugin-zksync",
```

Comment:
```suggestion
  "name": "@chainsafe/web3-plugin-zksync",
```

Reply:
repo is under web3 but use chainsafe org so its published on npm with chainsafe org, it prevents spam problem on npm